client/src/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import icon from "./icon.png";
+
+import "./header.css";
+
+
+
+function Header(){
+    return (
+    <div id = "head">
+        <div id = "insta">
+        <img src = {icon} id = "icon"></img>
+        <text id = "logo">Instaclone</text>
+        </div>
+        <button id = "camera"></button>
```

Comment:
ideally you should add img tag inside button tag and should not use background-img here

Reply:
okay i will do that